jit-compiler/src/codegen.rs

```diff
@@ -175,12 +179,15 @@ impl<'a, T: FieldElement> CodeGenerator<'a, T> {
                     type_: Some(type_),
                 },
             ) => {
-                let value = u64::try_from(value).unwrap_or_else(|_| unimplemented!());
-                match type_ {
-                    Type::Int => format!("ibig::IBig::from({value}_u64)"),
-                    Type::Fe => format!("FieldElement::from({value}_u64)"),
-                    Type::Expr => format!("Expr::from({value}_u64)"),
-                    _ => unreachable!(),
+                if *type_ == Type::Int {
+                    format_number(value)
+                } else {
+                    let value = u64::try_from(value).unwrap_or_else(|_| unimplemented!());
+                    match type_ {
+                        Type::Fe => format!("FieldElement::from({value}_u64)"),
+                        Type::Expr => format!("Expr::from({value}_u64)"),
+                        _ => unreachable!(),
+                    }
```

Comment:
I find having a single match for all types clearer

Reply:
so you would prefer doubling line 185?

Reply:
Yes

Reply:
Done.